tests/scripts/autoflake.sh

```diff
@@ -1,2 +1,2 @@
-#!/bin/bash
+#! /bin/bash
```

Comment:
there should not be a space between the hashbang and the path. Same comment in covreport.sh below.

Reply:
ok, I'll make the change